Content.Client/Construction/ConstructionButton.cs

```diff
@@ -27,12 +27,24 @@ protected override void Initialize()
         {
             base.Initialize();
 
+            AnchorLeft = 1.0f;
+            AnchorTop = 1.0f;
+            AnchorRight = 1.0f;
+            AnchorBottom = 1.0f;
+            MarginLeft = -110.0f;
+            MarginTop = -70.0f;
+            MarginRight = -50.0f;
+            MarginBottom = -50.0f;
+            Text = "Crafting";
             OnPressed += IWasPressed;
         }
 
         private void PerformLayout()
         {
-            Menu = new ConstructionMenu(_displayManager);
+            Menu = new ConstructionMenu(_displayManager)
+            {
+                Size = new Vector2(500.0f, 350.0f)
```

Comment:
Put this in the `ConstructionMenu` instead.

Reply:
Done